lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -574,16 +574,7 @@ LookupResult TypeChecker::lookupConstructors(DeclContext *dc, Type type,
   return lookupMember(dc, type, DeclBaseName::createConstructor(), options);
 }
 
-enum : unsigned {
-  /// Never consider a candidate that's this distance away or worse.
-  UnreasonableCallEditDistance = 8,
-
-  /// Don't consider candidates that score worse than the given distance
-  /// from the best candidate.
-  MaxCallEditDistanceFromBestCandidate = 1
-};
-
-static unsigned getCallEditDistance(DeclName writtenName,
+unsigned TypeChecker::getCallEditDistance(DeclName writtenName,
                                     DeclName correctedName,
```

Comment:
Nitpick: indentation should line up with (. Also, can this be a static method? It doesn't use any per-instance state from TypeChecker

Reply:
Thank you for the feedback. I've amended the commit to fix indentation and make it static.